microshift-okd/src/README.md

```diff
@@ -0,0 +1,27 @@
+## Build and Run Microshift upstream without subscription/pull-secret
+
+- building the container from src
+  > `cd microshift-okd/src && ./build.sh` 
+  - this script will:
+    1. replace microshift assets images to OKD  upstream images
+    1. will build microshift RPMs and repo based on current sources.
+    1. will build micrsoshift_okd bootc container based on `centos-bootc:stream9`
+    1. apply upstream customization  (see below)
+
+
+- running the container 
+  > `sudo podman run --privileged --rm --name microshift-okd -d microshift-okd`
+
+- connect to the container
+  > `sudo podman exec -ti microshift-okd /bin/bash`
+
+## configuration customization
+1. storage driver disabled (there is no lvms images upstream)
```

Comment:
topolvm is the upstream variant of lvms, and so should be added to the accepted values under .storage

Leveraging the config to turn off storage has some downsides:
1. community users won't get storage provisioning out of the box, unlike downstream
1. inverts the "opt-out" component design. A stable upstream cluster MUST be configured to opt out of storage. So it's really opting-in for a stable cluster - the opposite of downstream.
1. Future customers may be confused by the inverted meaning of .storage between upstream and downstream. Documentation could help, but the config is designed around an "opt-out" philosophy.
1. (Ignoring topolvm) the community build would be missing a key part of the microshift experience, which only complicates the out of box experience.

Reply:
I do not think we're ignoring CSI. Let's change the comment that it will be implemented in the next stages of this project. 
We should take it step-wise - the current change is already a great progress...